### ISSUE TITLE
Which files need to be hosted?

### ISSUE THREAD

**Issue by kennynaoh** [oncletom/tld.js#79]
Hello @oncletom ,
I am the member of [cdnjs](https://github.com/cdnjs/cdnjs) project.
We want to host this library.
I want to make sure with you whether `lib/rule.js` need to be grab or not.
And if any other dependency file need to be grab, please also tell me.
Thanks! for your help!

https://github.com/cdnjs/cdnjs/issues/8943
https://github.com/cdnjs/cdnjs/pull/8986

**Comment by oncletom** [comment]
Yeah you would need to include it and/or the browserified result of the `./index.js` entrypoint:

1. `npm run build` grabs the latest tld rules from a remote server and assemble them in a way tld.js can understand
2. the package is ready to be shipped/deployed to npm

If it is easier for you to grab the browserified build, I am happy to consider publishing it on npm (but not git).

**Comment by kennynaoh** [comment]
Thank you @oncletom

**Comment by kennynaoh** [comment]
I will add `lib/tld.js` and `lib/rule.js` to cdnjs.
Am I right?
Thanks!

**Comment by oncletom** [comment]
@kennynaoh well if you browserify `index.js` it shall suffice. It bundles the lib and the rules.

Unless you are trying to achieve something different?

**Comment by kennynaoh** [comment]
Sorry for late reply. I think it's better to combine these two file, `lib/tld.js` and `lib/rule.js`, what do you think? Thanks!

**Comment by oncletom** [comment]
@kennynaoh sorry for the late reply too 😓 

Actually v2.0.0 has been released and there are a bit more files now.
And the build process is gone.

I am unsure about how the cdnjs build process and how it builds browser compatible artefacts — if there is any change to do, feel free to ask.
